.codeclimate.yml

```diff
@@ -1,4 +1,5 @@
 exclude_paths:
 - vendor/**/*
 - ZeroBin/**/*
-- tests/**/* 
```

Comment:
I don't know exactly but if have to guess,then they both are pretty same.I will update them